src/pybind/mgr/dashboard/frontend/src/app/ceph/nfs/nfs-form/nfs-form.component.ts

```diff
@@ -181,9 +182,8 @@ export class NfsFormComponent extends CdForm implements OnInit {
   }
 
   volumeChangeHandler() {
-    this.pathChangeHandler();
-    const fs_name = this.nfsForm.getValue('fsal').fs_name;
-    this.getSubVolGrp(fs_name);
+    this.getSubVol();
+    this.checkVolumeValidation();
```

Comment:
There is redundancy involved..
volumeChangeHandler() -> getSubVol() -> setSubVolGrpPath -> checkVolumeValidation

And then again checkVolumeValidation.
Either we remove it from here or change our logic flow.